GOVERNANCE.md

```diff
@@ -0,0 +1,29 @@
+# Governance Policy
+
+This document provides the governance policy for the SpotBugs Project.
+
+## SpotBugs Core Term
+
+The SpotBugs Core Team maintain the spotbugs/spotbugs GitHub repository.
+
+The GitHub team for SpotBugs Team is `@spotbugs/core-devs`. Members of the SpotBugs Core Team have write access to the repository.
+
+## How we propose changes
+
+Both SpotBugs Core Team and SpotBugs user can propose changes to the SpotBugs project via GitHub pull requests. Refer to `.github/CONTRIBUTING.md` to know more detailed requirements for your proposal.
+
+Once a pull request gets two approvals from members of SpotBugs Core Team, pull request will be merged and shipped in the next release. Approver should not be the same as the author(s) of the change.
```

Comment:
> Approver should not be the same as the author(s) of the change.

This is contradictory to the current state / your comment: 

> It is mostly the same to the unwritten rule in this repo. Most of our merged PRs have two approvers, or one approver and another merger.

Currently if we have author from committers, we only want one more approval.
With this proposal, we will need two approvals - one more as today.

Reply:
Yes, this is a challenge as noted at https://github.com/orgs/spotbugs/teams/everyone/discussions/9?from_comment=4#discussion-9-comment-4

I think it's reasonable change for governance, how do you think?

Reply:
I don't think so. One can't force people to do this if they don't want or can't. I see mostly you and ThrawnCA active, and while I'm still there, I simply have no time to spent on reviews, especially if they are for features I'm not interested personally (lile gradle support etc). So by forcing two additional reviews you force me to do something I won't, and as a result I will be unhappy and feel me bad because on the one side I know your work waits for my review but on the other side I can't afford to spend reasonable amount of time for this (or don't want to review this).

Reply:
OK I understood. Thanks. I will remove the limitation later.